### ISSUE TITLE
Menu icons do not have labels - have tried to reinstall

### ISSUE THREAD

**Issue by HeatherRowe** [stevencohn/OneMore#854]
**Describe the bug**
no labels on the menu icons - new installation of OneMore
using OneNote version 2301 (Build 16026.20146) 
version of OneMore (64 bit) from the latest installer link

**To Reproduce**
I installed OneMore using the instructions on how to install
I have tried uninstalling, then shutting down computer, then reinstalling and shutting down computer, 
issue persists

**Expected behavior**
To see labels and be able to use the software (not sure if any of the buttons actually work currently)

**Screenshots**
three screenshots are attached
from Customize Ribbon menu - 
![OneMore - Customize Ribbon shows no labels](https://user-images.githubusercontent.com/89861769/215549833-6e84c613-4b43-42af-bdb5-5f7536155a4c.png)
image of drop-down menu (with no labels) -
![OneMore - drop down menus no labels](https://user-images.githubusercontent.com/89861769/215549836-117b0acc-8e5c-494e-ac0e-b875b9f2cca4.png)
image of section on Home Tab where OneMore icons appear - 
![OneMore - menu icons no labels](https://user-images.githubusercontent.com/89861769/215549839-392359b8-9731-433d-abaa-296410ad5862.png)

**Environment (if applicable):**
 - OneNote Version: O365, 64bit
 - OneMore Version - no idea, I can't find the About menu

**Additional context**
Add any other context about the problem here. Attach page XML file or the app log file as appropriate. You can find the log file here: %localappdata%\temp\OneMore.log
[OneMore.log](https://github.com/stevencohn/OneMore/files/10538313/OneMore.log)

**Comment by stevencohn** [comment]
Hi, I just updated my local installation of OneNote to 16026.20146 and installed the OneMore 5.5.0. It seems to work fine. Without more information, I'm not sure how to reproduce this. 

There is definitely a small window in which the OneMore interop connection to OneNote is broken and will result in the add-in falling into a broken state which manifests exactly as you're showing here with missing menu labels. In all cases that I've come across, restoring functionality is simply a matter of closing and reopening OneNote. Some times you have to kill it from Task Manager if the UI closes but the process still hangs. I've reported this to Microsoft but it's impossible to come up with a 100% reproducible test case to demonstrate it to them.

Have you tried an earlier version of OneMore? I can't imagine it would behave any differently but may be an interesting data point.

**Comment by HeatherRowe** [comment]
I had OneMore installed (and working) several months ago, but that was accidentally uninstalled by my IT group when they applied an update to my machine.   
I will look into installing an earlier version again to see if that makes any difference, and will report back on anything I find.

Unfortunately, closing and reopening OneNote, even completely shutting down the machine and restarting, has not fixed this yet.

**Comment by stevencohn** [comment]
"IT group." 'nuf said.

**Comment by JRPX33** [comment]
Hey now, be nice to us IT guys!

But seriously, I also saw this a few times on the previous version.    Seemed to be related to low memory/system stress issues -- when other stuff (AV, Chrome, etc...) was making OneNote slow to respond I'd see this happen.  Closing and reopening always fixed it.

Since I'm an evil IT guy and can get some process monitor or process explorer traces next time to get more data :-)

Jeff
